APMrover2/mode_loiter.cpp

```diff
@@ -49,7 +49,11 @@ void ModeLoiter::update()
             yaw_error_cd = wrap_180_cd(_desired_yaw_cd - ahrs.yaw_sensor);
             _desired_speed = -_desired_speed;
         }
-
+        if(g2.loit_type == 2){
```

Comment:
this could be included in the above if,  if ((fabsf(yaw_error_cd) > 9000 && g2.loit_type == 0) || g2.loit_type == 2)